app/src/ui/repositories-list/group-repositories.ts

```diff
@@ -132,8 +132,10 @@ export function makeRecentRepositoriesGroup(
   for (const id of recentRepositories) {
     const repository = repositories.find(r => r.id === id)
     if (repository !== undefined) {
-      const existingCount = names.get(repository.name) || 0
-      names.set(repository.name, existingCount + 1)
+      const alias = repository instanceof Repository ? repository.alias : null
+      const name = alias ?? repository.name
+      const existingCount = names.get(name) || 0
+      names.set(name, existingCount + 1)
```

Comment:
This list of names is used to detect duplicates in the `Recents` group, to know when we need to disambiguate showing the owner as a prefix (e.g. `github/desktop`).
In this change I'm just adding the alias instead of the repo name, when available.